tests/MongoDB.Driver.Core.Tests/Core/Authentication/ScramSha256AuthenticatorTests.cs

```diff
@@ -302,6 +308,54 @@ public void Constructor_should_throw_an_ArgumentNullException_when_credential_is
             subject._cache()._cacheKey().Should().NotBe(null);
             subject._cache()._cachedEntry().Should().NotBe(null);
         }
+
+#if !NETCOREAPP1_1
```

Comment:
NETCOREAPP1_1 doesn't allow using `Thread.CurrentThread.CurrentCulture`

Reply:
By the same reason, I didn't add a new BeforeAfterTestAttribute

Reply:
I'm not sure I really like `[BeforeAfterTestAttribute]` anyway. It's too easy to forget to add the attribute to the test method.

Reply:
It's also easy just to not add any logic to reset this value after the test. I think it's a matter of habit.

Reply:
The least likely to be forgotten would be a one-liner:

    using (TemporarySomething.Set(...))
    {
        // test run with something temporary
    }

Reply:
No changes needed here though. But in the future I would prefer a more reliable one-line approach than a `BeforeAfterTestAttribute`.

Reply:
My point that this line `using (TemporarySomething.Set(...))` can be simply forgotten too as well as an attribute. For me, it's more like this is a question to the developer and his code habits rather than real disadvantages.
Let's discuss it later :)

Reply:
My point is that if the setting and resetting of a temporary mode are ONE line of code it's harder to forget the resetting.

No changes needed here though.